spec/helpers/gov_one_helper_spec.rb

```diff
@@ -4,28 +4,36 @@
   describe '#login_uri' do
     subject(:login_uri) { helper.login_uri }
 
-    it 'returns a URI object with the correct host and path' do
-      expect(login_uri.host).to eq 'oidc.test.account.gov.uk'
-      expect(login_uri.path).to eq '/authorize'
-    end
-
     it 'encodes the authorize endpoint params' do
-      expect(login_uri.query).to start_with 'response_type=code&scope=email+openid&client_id='
-      expect(login_uri.query).to end_with '&redirect_uri=http%3A%2F%2Frecovery.app%2Fusers%2Fauth%2Fopenid_connect%2Fcallback'
+      expect(login_uri).to start_with 'https://oidc.test.account.gov.uk/authorize?response_type=code&scope=email+openid&client_id='
+      expect(login_uri).to end_with 'redirect_uri=http%3A%2F%2Frecovery.app%2Fusers%2Fauth%2Fopenid_connect%2Fcallback'
     end
   end
 
   describe '#logout_uri' do
     subject(:logout_uri) { helper.logout_uri }
 
-    it 'returns a URI object with the correct host and path' do
-      expect(logout_uri.host).to eq 'oidc.test.account.gov.uk'
-      expect(logout_uri.path).to eq '/logout'
+    it 'encodes the logout endpoint params' do
+      expect(logout_uri).to start_with 'https://oidc.test.account.gov.uk/logout?id_token_hint&state='
+      expect(logout_uri).to end_with '&post_logout_redirect_uri=http%3A%2F%2Frecovery.app%2Fusers%2Fsign_out'
     end
+  end
 
-    it 'encodes the logout endpoint params' do
-      expect(logout_uri.query).to start_with 'id_token_hint&state='
-      expect(logout_uri.query).to end_with '&post_logout_redirect_uri=http%3A%2F%2Frecovery.app%2Fusers%2Fsign_out'
+  describe '#login_button' do
+    subject(:login_button) { helper.login_button }
+
+    it 'returns a link to the login uri' do
```

Comment:
The description of this test is not actually what it checks.

Reply:
[36ee42d](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/917/commits/36ee42d12e27394f9c0a92bce46dc792fed49912)